src/txn/txn_timestamp.c

```diff
@@ -579,14 +579,15 @@ set:	__wt_writelock(session, &txn_global->rwlock);
 }
 
 /*
- * __wt_timestamp_validate --
- *	Validate a timestamp to be not older than the global oldest and global
- *	stable and running transaction commit timestamp and running transaction
- *	prepare timestamp.
+ * __wt_commit_timestamp_validate --
+ *	Validate a timestamp to be not older than running transaction commit
+ *	timestamp and running transaction prepare timestamp. Validate a durable
+ *	timestamp to be not older than the global oldest and global stable
+ *	timestamp.
  */
 int
-__wt_timestamp_validate(WT_SESSION_IMPL *session, const char *name,
-    wt_timestamp_t ts, WT_CONFIG_ITEM *cval, bool compare_stable)
+__wt_commit_timestamp_validate(WT_SESSION_IMPL *session, const char *name,
```

Comment:
While you are changing the name, a prefix of `__wt_txn_XXX` is more consistent with other functions in the file.